ocs_ci/utility/retry.py

```diff
@@ -74,3 +74,55 @@ def wrapper(*args, **kwargs):
         return wrapper
 
     return decorator
+
+
+def retry_until_exception(
```

Comment:
Instead of creating a separate function can we handle this in the existing retry function? One way is to add a new arg until_exception defaulting to False and return True only if until_exception.
I am not sure if it can cause any regression to the existing function but it is something that we can try as it reduces the code redundancy.

Reply:
Retry serves purpose of "retry until we dont see the given exception" and retry until exception is we want to retry until the exception is occured. Its two different purpose and managing it in single function doesnt sound good to me even though it reduces the number of lines of code.